### PR TITLE
chore: update contributors

### PR DESCRIPTION
**Pull request by Isaiah-Hamilton**
update careers page contributors

### CODE REVIEW

File: apps/www/data/career.json [hunk 1]
```diff
@@ -55,7 +55,6 @@
     "mikqi",
     "florian-lefebvre",
     "m4salah",
-    "carlosstenzel",
```
Comment (dshukertjr) [inline apps/www/data/career.json:58]: Who is this BTW?

Comment (Isaiah-Hamilton) [reply apps/www/data/career.json:58]: https://github.com/carlosstenzel

Comment (dshukertjr) [reply apps/www/data/career.json:58]: Is the reason for removing because we can only have so many contributors shown in the page?

Comment (dshukertjr) [reply apps/www/data/career.json:58]: Just asking, because I don't know how the website works!

Comment (Isaiah-Hamilton) [reply apps/www/data/career.json:58]: The figma design had a max of 18, so I just went to off of that. but I can add one more in the top right corner without removing anyone.

File: apps/www/pages/careers/index.tsx [hunk 2]
```diff
@@ -65,6 +75,7 @@ export const getServerSideProps: GetServerSideProps = async () => {
 }
 
 const CareerPage: NextPage = ({ jobs, contributors }: any) => {
+  console.log(contributors)
```
Comment (thorwebdev) [inline apps/www/pages/careers/index.tsx:78]: remove?

**Review by thorwebdev — APPROVED** [review]
LGTM 🔥

**Review by dshukertjr — COMMENTED** [review]
Love it! If I may, could we shuffle the order so that we see more varieties of contributors on mobile as well?

**Comment by Isaiah-Hamilton** [comment]
> Love it! If I may, could we shuffle the order so that we see more varieties of contributors on mobile as well?

Done!

**Pull request merged**